notebooks/estimate_resource_usage.ipynb

```diff
@@ -121,7 +121,7 @@
    "source": [
     "## Set up testing workflows\n",
     "\n",
-    "in the `src/cromwell_manager/test/data/` folder are four WDL files pointing to google storage locations that each contain a different number of fastq reads: 10,000, 100,000, 1,000,000, and 10,000,000. We will use a monitoring script to run each of these workflows, and then calculate the resource usage (memory and disk) of each file throughout each of the run tasks"
+    "in the `cromwell_manager` repository are four WDL files pointing to google storage locations that each contain a different number of fastq reads: 10,000, 100,000, 1,000,000, and 10,000,000. We will use a monitoring script to run each of these workflows, and then calculate the resource usage (memory and disk) of each file throughout each of the run tasks"
```

Comment:
I think including a link to the repository would be helpful here since it's not a repo in the same organization as this one.

Reply:
Great idea, thanks! 

I added a link to the repo. I'm tempted not to add the data files in this case since they are easily accessible (and automatically downloaded within the notebook) from github. 

Because of the way the manager repo loads files, the wdls are just stored in memory and never hit the disk.

Reply:
Ok! Personally I think it's clearer to have the wdl files being analyzed in the same repo as this notebook, but as long as it's clear where those files are coming from I think it's okay!